xbmc/video/tags/VideoTagLoaderFFmpeg.cpp

```diff
@@ -132,8 +139,44 @@ bool CVideoTagLoaderFFmpeg::HasInfo() const
   return avtag != nullptr;
 }
 
-CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::Load(CVideoInfoTag& tag, bool)
+CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::Load(CVideoInfoTag& tag,
+                                                    bool, std::vector<EmbeddedArt>* art)
+{
+  if (m_item.IsType(".mkv"))
+    return LoadMKV(tag, art);
+  else if (m_item.IsType(".mp4"))
+    return LoadMP4(tag, art);
+  else
+    return CInfoScanner::NO_NFO;
+
+}
+
+CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::LoadMKV(CVideoInfoTag& tag,
+                                                       std::vector<EmbeddedArt>* art)
 {
+  // embedded art
+  for (size_t i = 0; i < m_fctx->nb_streams; ++i)
+  {
+    AVDictionaryEntry* avtag=nullptr;
```

Comment:
missing spaces

Reply:
superfluous assignment so addressed that instead.